inspire_dojson/journals/rules.py

```diff
@@ -44,71 +44,164 @@ def issn(self, key, value):
     b = value.get('b', '').lower()
     if 'online' == b:
         medium = 'online'
-        comment = ''
     elif 'print' == b:
         medium = 'print'
-        comment = ''
     elif 'electronic' in b:
         medium = 'online'
-        comment = 'electronic'
     elif 'ebook' in b:
         medium = 'online'
-        comment = 'ebook'
     elif 'hardcover' in b:
         medium = 'print'
-        comment = 'hardcover'
     else:
         medium = ''
```

Comment:
The previous implementation was not correct. Here's what's actually in `022__b`:
```
    103 Online
     79 Print
      7 electronic
      6 print
      3 online
      1 Print (Russian)
      1 printed
      1 Online (Russian
      1 Online (English)
      1 e-only
```

Reply:
Fixed.